registry.json

```diff
@@ -1,4 +1,19 @@
 [
+    {
+        "ticker": "RELAY",                                                      
+        "poolId": "6c1c1fc2b1e3e1ac056f6dd4dd4c3b78e66bd01896ca6e98a4d2431e",
+        "social": {                                                            
+            "twitter": "",
+            "website": "https://relaypool.com",
+            "facebook": " ",  
+            "instagram": " ",
+            "youtube": " ",
+            "reddit": "Relay_ADA_Pool"
+        },
+        "telegram": "MVS_RELAY",                              
+        "operatorName": "MVS",		                                   
```

Comment:
Please provide you real first name or full name here.  If your rather not, leave the field empty.